### PR TITLE
adding ac profile uploading

### PR DESCRIPTION
**Pull request by quasiben**
PR adding anaconda cluster profile uploading

### CODE REVIEW

File: binstar_client/inspect_package/profile.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import yaml
+import os
+#
+# self.binstar.upload(self.username, self.packagename,
```
Comment (malev) [inline binstar_client/inspect_package/profile.py]: please remove commented code

**Comment by quasiben** [comment]
apologies

File: binstar_client/utils/detect.py [hunk 2]
```diff
@@ -76,6 +78,13 @@ def is_r(filename):
     else:
         log.debug("This not is an R package (expected .tgz, .tar.gz).")
 
+def is_cluster_profile(filename):
```
Comment (malev) [inline binstar_client/utils/detect.py]: I feel like we are gonna need more proof that this is a `cluster-profile` file. We are also gonna be uploading environments to AS and they also have `yaml` extension.
How about checking that it has a "provider", "node_id" and "number of nodes" something like that?

Comment (quasiben) [inline binstar_client/utils/detect.py]: I do that in the `inspect_profile_package` function:  https://github.com/Anaconda-Server/anaconda-client/blob/feature/profile_upload/binstar_client/inspect_package/profile.py#L9.  Butt we are also going to be adding another key/value pair indicating api version and that it's an anaconda cluster profile

Comment (malev) [inline binstar_client/utils/detect.py]: sounds good to me

**Comment by quasiben** [comment]
Closing

**Pull request closed without merging**